core/src/main/java/org/pentaho/di/core/util/StringUtil.java

```diff
@@ -613,4 +613,46 @@ public static String safeToLowerCase( final Object obj ) {
       return obj.toString().toLowerCase();
     }
   }
+
+  /**
+   * Removes all instances of the specified character from the start of the given {@link String}.
+   *
+   * @param s the {@link String} to trim
+   * @param c the character to remove from the {@link String}
+   * @return a new string with all instances of the specified character removed from the start
+   */
+  public static String trimStart( final String s, char c ) {
```

Comment:
Suggest you pattern this after Const.ltrim - this is a lot of substring operations which create temporary strings and doesn't perform well at all. Something more like this:

public static String ltrim( String source, char c ) {
    if ( source == null ) {
      return null;
    }
    int from = 0;
    int len = source.length();
    while ( from < len && ( source.charAt( from ) == c ) ) {
      from++;
    }
    return source.substring( from );
  }